src/app/view/components/dot-dialog/dot-dialog.component.spec.ts

```diff
@@ -175,6 +179,17 @@ describe('DotDialogComponent', () => {
                 expect(footer === null).toBe(false);
             });
 
+            it('should append component to body', () => {
+                component.appendToBody = true;
+                hostComponent.show = false;
+                hostFixture.detectChanges();
+                hostComponent.show = true;
```

Comment:
why is set first to false and later to true?

Reply:
to simulate the close and open of the dialog with the new property `appentToBody` set, and fire this https://github.com/dotCMS/core-web/pull/1124/commits/b96a8fc325eb6e4983d7755d8b4aec1bf354f4df#diff-6dff7f46d47b2129b3c8f59a685f15a3L76